src/components/organisms/shared/eventsDrawer.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+
+import { useNavigate, useParams } from "react-router-dom";
+
+import { EventsTable } from "../events";
+import { DrawerName } from "@src/enums/components";
+
+import { IconButton } from "@components/atoms";
+import { Drawer } from "@components/molecules";
+
+import { Close } from "@assets/image/icons";
+
+export const EventsDrawer = () => {
+	const navigate = useNavigate();
+	const [isOpen, setIsOpen] = useState(false);
+	const { projectId, triggerId } = useParams();
+
+	useEffect(() => {
+		setIsOpen(location.pathname.includes("/events"));
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [location.pathname]);
```

Comment:
Don't decide if it's open by the pathname.
Create another page of triggers/connections with the drawer always open and use it.

Reply:
Let's think about it together